mentions-generator/src/main/java/de/brandwatch/minianalytics/mentiongenerator/MentionsGeneratorApplication.java

```diff
@@ -17,9 +19,16 @@ public static void main(String[] args) {
     @Autowired
     private Producer producer;
 
+    @Autowired
+    private Consumer consumer;
+
+    @Autowired
+    private QueryRepository queryRepository;
+
     @PostConstruct
-    public void sendMessagesOnKafka() {
-        producer.send("Waz up?");
+    public void matchMentionsOnQueries(){
```

Comment:
It's still not clear to me what this is for

Reply:
Artifact of old tests. Removed.